twitch.py

```diff
@@ -1700,7 +1700,7 @@ async def get_live_streams(self, game: Game, *, limit: int = 30) -> list[Channel
             })
         )
         return [
-            Channel.from_directory(self, stream_channel_data["node"])
+            await Channel.from_directory(self, stream_channel_data["node"])
```

Comment:
The core idea of building channels from directory was not having to fetch any additional data for each channel, because it delays the refresh loop for quite a bit, without much benefit.